frontend/web/components/ExternalResourcesTable.tsx

```diff
@@ -30,6 +47,177 @@ const ExternalResourcesTable: FC<ExternalResourcesTableType> = ({
       toast('External resources was deleted')
     }
   }, [isDeleted])
+  return (
+    <Row className='list-item' key={externalResource?.id}>
+      <Flex className='table-column mt-1'>
+        <Row className='font-weight-medium'>
+          {externalResource?.type === 'GITHUB_ISSUE'
```

Comment:
This is tied to GitHub unnecessarily.

 Why not just add the following in Constants
 
 ```
resourceTypes: {
GITHUB_ISSUE: "GitHub Issue"
}
```

then 

```
 <Row className='font-weight-medium'>
 Constants.resourceTypes[externalResource?.type]]
```